src/devices/src/virtio/net/device.rs

```diff
@@ -34,6 +34,15 @@ use virtio_gen::virtio_net::{
 };
 use vm_memory::{ByteValued, Bytes, GuestAddress, GuestMemoryError, GuestMemoryMmap};
 
+pub enum FrontendError {
```

Comment:
No need for `pub`. This error type is completely swallowed by `write_frame_to_guest` which converts it (sort of) into a `bool`.